modules/gitops-release-notes-1-12-4.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-12-4_{context}"]
+= Release notes for {gitops-title} 1.12.4
+
+{gitops-title} 1.12.4 is now available on {OCP} 4.12, 4.13, 4.14, and 4.15.
+
+[id="errata-updates-1-12.4_{context}"]
+== Errata updates
+
+[id="rhsa-2024:4163-gitops-1-12-4-security-update-advisory_{context}"]
+=== RHSA-2024:4163 - {gitops-title} 1.12.4 security update advisory
+
+Issued: 2024-06-27
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-2024:4163[RHSA-2024:4163]
```

Comment:
make sure this becomes open to unauthorized users when you merge

Reply:
https://github.com/openshift/openshift-docs/pull/78107#discussion_r1660484239